lib/resque-serializer/configuration.rb

```diff
@@ -0,0 +1,13 @@
+module Resque
+  module Plugins
+    module Serializer
+      class Configuration
+        attr_accessor :generate_mutex
```

Comment:
It could be safer to use `singleton` pattern in here to not to face thread issues:
`include Singleton`
Ref: https://ruby-doc.org/stdlib-2.5.1/libdoc/singleton/rdoc/Singleton.html